src/peer_conn_protocol.rs

```diff
@@ -31,5 +32,5 @@ pub fn send_p2p_handshake(
     )
     .unwrap();
 
-    Uuid::new_v4()
+    Ok(Uuid::new_v4())
```

Comment:
This part of the code is still in progress- I've changed it up so that the handshake gets sent and nothing panics but it doesn't work- this uuid isn't recorded anywhere so we don't send the peer any packets. Just as well since they would be nonsense to the peer at this point since we don't have a way to translate it.